src/components/RegionInput.js

```diff
@@ -20,45 +20,63 @@ export const RegionInput = ({
   const pathNamesColon = pathNames.map((name) => name + ":");
   const pathsWithRegion = [];
 
+  // Store possible options from bed file and match them with their respective tracks
+  let optionToTrack = {};
+
   if (regionInfo && !isEmpty(regionInfo)) {
     // Stitch path name + region start and end
     for (const [index, path] of regionInfo["chr"].entries()) {
       pathsWithRegion.push(
         path + ":" + regionInfo.start[index] + "-" + regionInfo.end[index]
       );
     }
+
+    // populate optionToTrack with paths with region
+    pathsWithRegion.forEach((path, i) => optionToTrack[path] = regionInfo.tracks[i]);
+
     // Add descriptions
     pathsWithRegion.push(...regionInfo["desc"]);
+
+    // populate optionToTrack with paths with description as keys
+    regionInfo.desc.forEach((desc, i) => optionToTrack[desc] = regionInfo.tracks[i]);
   }
 
   // Autocomplete selectable options
   const displayRegions = [...pathsWithRegion, ...pathNamesColon];
 
-  
+  console.log("displayRegions", displayRegions);
   return (
     <>
       <Autocomplete
         disablePortal
         freeSolo // Allows custom input outside of the options
-        getOptionLabel={(option) => option.title || option.toString()}
+        autoHighlight
+        getOptionLabel={(option) => option.label || option.toString()}
         value={region}
         inputValue={region}
         data-testid="autocomplete"
         id="regionInput"      
 
         onInputChange={(event, newInputValue) => {
-          handleRegionChange(newInputValue);
+          // If an option is selected, should have a match in optionToTrack
+          if (event.target.textContent in optionToTrack) {
+            // also pass tracks associated with the option
+            handleRegionChange(newInputValue, optionToTrack[event.target.textContent]);
+          } else {
+            handleRegionChange(newInputValue, null);
+          }
         }}
 
         options={displayRegions}
         renderInput={(params) => (
           <TextField
             {...params}
             label="Region"
-            style={{zIndex:'-100'}}
+            //style={{zIndex:'-100'}}
```

Comment:
I think we need this to keep the "Region" label text from drawing over the BED file dropdown when the BED file dropdown is open. I tested your branch here and it seems to reintroduce that bug. Uncommenting this line fixes that but then the region dropdown doesn't pop open when clicked.

If we have to take this out to get the region dropdown to pop open when clicked (because e..g it already is popping open but it is rendering behind the page somehow) then we have to find another way to keep the "Region" label from drawing over the BED file dropdown.